geekodoc/rng/2_5.2/geekodoc-v2.rnc

```diff
@@ -135,22 +135,20 @@ div {
      "DIA" |
      ## Format of the dia tool
      "dia" |
+     ## Format of the ditaa tool
+     "DITAA" |
+     "ditaa" |
      ## Encapsulated PostScript
      "EPS" |
      ## Encapsulated PostScript
      "eps" |
-     ## Format of the xfig tool
-     "FIG" |
-     ## Format of the xfig tool
-     "fig" |
+     ## Format of the DITAA tool
+     "DITAA" |
+     "ditaa" |
```

Comment:
Why does this occurs twice? It's already in line 138-140.

Reply:
Indeed that should be fixed before merging. However, on the general point I made in the description regarding listing all of those input formats ... what do you say?